build.properties

```diff
@@ -1,9 +1,9 @@
 VERSION=4.1.3
-MINECRAFT_VERSION=1.19.4
-FORGE_VERSION=45.1.0
+MINECRAFT_VERSION=1.20.1
+FORGE_VERSION=47.2.0
 MAPPINGS_CHANNEL=official
-MAPPINGS_VERSION=1.19.4
-JEI_VERSION=13.1.0.15
+MAPPINGS_VERSION=1.20.1
+JEI_VERSION=15.3.0.4
 # Removed for now as no 1.19 versions exist
 TETRA_FILE=
 MUTIL_FILE=
```

Comment:
we should probably reimplement tetra support... maybe...
this is not related to the PR, please ignore